### ISSUE TITLE
New App Suggestions

### ISSUE THREAD

**Issue by Luigi9606** [InsideTechnologiesSrl/DefenderBug#1]
Hi, thanks a lot for your work on this! I wonder whether you'd mind adding lines for Adobe Reader, Sibelius (music notation software) and VLC. I haven't tested this in our environment yet but are you able to confirm that this will reinstate the shortcuts of multi-region versions of MS Office? I've heard reports that the script released by MS only works with en-US. Thank you

**Comment by InsideTechnologiesSrl** [comment]
I can confirm that works with all languages.
Adobe is now added but for the other, we need the path

**Comment by kylian-002** [comment]
Could Pulse secure also be added?
"pulse Secure" = "C:\Program Files (x86)\Common Files\Pulse Secure\JamUI\Pulse.exe"

To add applications is it just a matter of adding those lines to the file? As i did try that with Pulse secure but got a error
'Test-Path : Cannot bind argument to parameter 'Path' because it is an empty string.'

**Comment by InsideTechnologiesSrl** [comment]
Yes, this is the only required point....unless the icon is not already present or there's something else into the app.
